docs/faq.rst

```diff
@@ -198,7 +198,7 @@ Quick example: ::
     # or '\U0001f44d' or '👍'
     await message.add_reaction(emoji)
 
-In case you want to use emoji that come from a message, you already get their code points in the content without needing
+In case you want to use emojis that come from a message, you already get their code points in the content without needing
```

Comment:
Emoji is either countable or uncountable, so the previous sentence was fine. Since Japanese doesn't really have plurals I prefer the uncountable version (same with "sushi" vs "sushis" or "kanji" vs "kanjis") even though the countable version is technically valid too.